### PR TITLE
Agents app

### PR DESCRIPTION
**Pull request by dmitrii-khizbullin**

### CODE REVIEW

**Comment by Obs01ete** [comment]
@lightaime PR is ready, pls review and merge

**Review by lightaime — COMMENTED** [review]
Awesome! I like it. Leave some minor comments. Please merge it after you fix them. Also please remember to fix these `flake8` issues: https://github.com/lightaime/camel/actions/runs/4585128885/jobs/8097075241?pr=37. Thanks!

File: apps/agents/agents.py [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+            - Progress bar contents.
+    """
+
+    if state.session is not None:
+        print("Double click")
+        return {}  # may fail
+
+    try:
+        session = RolePlaying(assistant, user, original_task,
+                              with_task_specify=True, with_task_planner=False)
```
[the remark this answers is not in the document]
Comment (Obs01ete) [reply apps/agents/agents.py:146]: Will put it in the backlog

File: apps/agents/agents.py [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+            gr.Markdown(
+                "## CAMEL: Communicative Agents for \"Mind\" Exploration"
+                " of Large Scale Language Model Society\n"
+                "Github repo: [https://github.com/lightaime/camel]"
+                "(https://github.com/lightaime/camel)")
+    with gr.Row():
+        with gr.Column(scale=9):
+            original_task_ta = gr.TextArea(
+                label="Give me a preliminary idea (EDIT ME)",
+                value=default_task, lines=1, interactive=True)
```
[the remark this answers is not in the document]
Comment (Obs01ete) [reply apps/agents/agents.py:325]: Put where? To the GUI as a static text?

File: apps/agents/agents.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (lightaime) [reply apps/agents/agents.py:145]: Sounds good.

File: apps/agents/agents.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (lightaime) [reply apps/agents/agents.py:325]: Yes, we can do that. Giving more explanations would be helpful for users

**Pull request merged**